### PR TITLE
reverts super_admin as a spree role.  correctly defaces the admin/user f...

### PR DESCRIPTION
**Pull request by jsqu99**
...orm to include the super_admin checkbox

forgot to add this to the commit message: prevents double-admin roles.

### CODE REVIEW

File: app/models/spree/user_decorator.rb [hunk 1]
```diff
@@ -15,7 +17,7 @@
   }
 
   before_save :ensure_at_least_one_super_admin_exists
-  before_save :ensure_super_admin_is_admin
+  after_save :ensure_super_admin_is_admin
```
Comment (mscottford) [inline app/models/spree/user_decorator.rb:20]: Should this stay `before_save`?

Comment (jsqu99) [inline app/models/spree/user_decorator.rb:20]: I had sent this in a skype re: this subject:

how strongly do you feel about https://github.com/railsdog/spree_landlord/blob/master/app/models/spree/user_decorator.rb#L34

in the admin app, if i select 'admin' & 'super_admin', it creates two 'admin' roles and one 'super_admin' b/c of this method

i tried changing the 'if' to     if self.super_admin? && !self.has_spree_role?(:admin)
, but the role is still in the process of being added, i believe

oh, i should try an after_save instead

Comment (mscottford) [inline app/models/spree/user_decorator.rb:20]: Okay.

**Comment by mscottford** [comment]
Looks good.

**Pull request merged**